src/sections/2022/section-1.mdx

```diff
@@ -0,0 +1,7 @@
+import ScreenBlock from "../../components/2022/screen-block"
+
+<div id="section-1">
```

Comment:
Could we make the ```DIV``` a section to be more accessible to screen readers?
[vid](https://www.youtube.com/watch?v=ULdkpU51hTQ)
```
<section aria-labeledby="Respondents" id="section-1">
  <ScreenBlock>
    <h2 id="Respondents">The Respondents</h2>
  </ScreenBlock>
</div>
```

Reply:
Same with the other sections below too...???

Reply:
Not something worth adjusting. This code gets scrapped in #339